File: September Challenge/Week 3/Java/Day_15/LengthOfLastWord.java
// Problem : https://leetcode.com/problems/length-of-last-word/
// Solution:

class Solution {
    public int lengthOfLastWord(String s) {
        int i=s.length()-1;
        int ans =0;
        while(i>=0 && s.charAt(i)==' '){
            i--;
        }
        while(i>=0 && s.charAt(i)!=' '){
            ans++;
            i--;
        }
        return ans;
    }
}

File: September Challenge/Week 3/C++/Day_15/LengthOfLastWord.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        if(s.size()==0){
            return 0;
        }
        int count=0;
        for(int i=s.length()-1;i>=0;i--){
            if(count!=0 && isspace(s.at(i))){
                break;
            }
            if(!isspace(s.at(i))){
                count++;
            }
        }
        return count;
    }
};
